package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "taskcluster-events",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "test": "mocha test/*_test.js",
+    "lint": "eslint src/*.js test/*.js",
+    "pretest": "yarn lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/taskcluster/taskcluster-events.git"
+  },
+  "dependencies": {
+    "debug": "^3.1.0",
+    "lodash": "^4.6.1",
+    "slugid": "^1.1.0",
+    "taskcluster-client": "^3.1.1",
+    "taskcluster-lib-app": "^3.0.0",
```

Comment:
hmm, maybe added `taskcluster-lib-api`, it's non-trivial but we can probably abuse it to return server-sent-events by not using the `res.reply` method... That way we can also document the end-point, we probably can't use the schema validation for anything, but at-least we can write docs for the end-point...

Obviously, said docs should say that the end-point doesn't not return JSON, but streams server-sent-events... :)
Hmm, something to think about and maybe try... I'll admit tc-lib-api is not aimed at writing things other than JSON end-points, but we do abuse it for artifacts too... it's not plausibly not crazy to stream things from such and end-point.

Reply:
Even I had a similar thought while writing the loader for the server (`takcluster-lib-app`).  But i decided maybe keeping an api connection open for long isn't the best option.

With this,we will need 4 endpoints for bind / unbind / pause / resume requests from the client. And then have some sort of common interface in the back which all the apis communicate.
Also a connect endpoint.